spaces/domains.json

```
@@ -320,4 +320,5 @@
   "vote.agdeportes.com": "agdeportes.eth",
   "vote.meebitsdao.com": "meebitsdao.eth",
   "vote.dopeshibas.fun": "dopeshibas.eth"
+  "vote.nftorbit.io": "nftorbit.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.dopeshibas.fun": "dopeshibas.eth",
  "vote.nftorbit.io": "nftorbit.eth"
```